### PR TITLE
Nit 798 ldap automate ldap uplift

### PR DESCRIPTION
**Pull request by georgepstaylor**
- Adding RBAC tag
- Fixing issue with log driver
- Fixing issue with cluster arn output

### CODE REVIEW

File: .github/workflows/build-push-automation-image.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: Docker build
         run: |
-            docker build --no-cache -t ${{ vars.CORE_SHARED_SERVICES_ACCOUNT }}.dkr.ecr.${{ vars.AWS_REGION }}.amazonaws.com/delius-core-ldap-automation-ecr-repo:$(echo ${{ steps.BumpVersionAndPushTag.outputs.new_tag }} | sed 's/[^a-zA-Z0-9.]/-/g') ./operational-automation
+            docker build --no-cache --build-arg="RBAC_TAG=${{vars.RBAC_TAG}}" -t ${{ vars.CORE_SHARED_SERVICES_ACCOUNT }}.dkr.ecr.${{ vars.AWS_REGION }}.amazonaws.com/delius-core-ldap-automation-ecr-repo:$(echo ${{ steps.BumpVersionAndPushTag.outputs.new_tag }} | sed 's/[^a-zA-Z0-9.]/-/g') ./operational-automation
```
Comment (kylehodgetts) [inline .github/workflows/build-push-automation-image.yml:37]: `vars.CORE_SHARED_SERVICES_ACCOUNT` and `vars.AWS_REGION` aren't variable. Perhaps we should follow the lead of other workflows and store these as secrets?

File: .github/workflows/operational-automation.yml [hunk 2]
```diff
@@ -28,6 +28,15 @@ jobs:
                 aws-access-key-id: ${{ secrets.CICD_ACCESS_KEY }}
                 aws-secret-access-key: ${{ secrets.CICD_SECRET_ACCESS_KEY }}
                 aws-region: "eu-west-2"
+        #ci-cd-member user cant do this yet
```
Comment (kylehodgetts) [inline .github/workflows/operational-automation.yml:31]: Are we leaving this comment in?

File: .github/workflows/operational-automation.yml [hunk 3]
```diff
@@ -63,7 +76,10 @@ jobs:
...
-                aws ecs run-task --cluster ${{ vars.CLUSTER_ARN }} --task-definition $task_definition_arn --network-configuration "awsvpcConfiguration={subnets=${{ steps.get-ids.outputs.SUBNET_IDS }},securityGroups=${{ steps.get-ids.outputs.SEC_GROUPS }},assignPublicIp=DISABLED}" --launch-type FARGATE --count 1
+                aws ecs run-task --cluster ${{ steps.set-cluster-arn.outputs.CLUSTER_ARN }} --task-definition $task_definition_arn --network-configuration "awsvpcConfiguration={subnets=${{ steps.get-ids.outputs.SUBNET_IDS }},securityGroups=${{ steps.get-ids.outputs.SEC_GROUPS }},assignPublicIp=DISABLED}" --launch-type FARGATE --count 1
           - name: Delete Task Definition
             run: |
-                aws ecs deregister-task-definition --task-definition ${{steps.run-task.outputs.TASK_DEF_ARN}}
\ No newline at end of file
+                aws ecs deregister-task-definition --task-definition ${{steps.run-task.outputs.TASK_DEF_ARN}}
+          - name: Tail aws logs
+            run: |
+                aws logs tail --follow --since 1m /ecs/ldap-automation/${{ github.run_id }}
```
Comment (kylehodgetts) [inline .github/workflows/operational-automation.yml:85]: new line

**Review by kylehodgetts — CHANGES_REQUESTED** [review]

File: .github/workflows/operational-automation.yml (hunk 2 above)
Comment (georgepstaylor) [reply .github/workflows/operational-automation.yml:31]: Yeah I want to add the permissions in at some point

**Review by kylehodgetts — APPROVED** [review]

**Pull request merged**